torch/csrc/jit/frontend/schema_matching.h

```diff
@@ -39,25 +41,22 @@ TORCH_API bool convertibleToList(
     const TypePtr& type,
     const TypePtr& list_type_);
 
-TORCH_API Value* emitBuiltinCall(
-    const SourceRange& loc,
-    Graph& graph,
-    Symbol name,
-    at::ArrayRef<NamedValue> args,
-    at::ArrayRef<NamedValue> kwargs,
-    const c10::optional<NamedValue>& self = c10::nullopt);
-
 TORCH_API c10::optional<size_t> findInputWithName(
     const std::string& name,
     at::ArrayRef<NamedValue> kwargs);
 
 // applies implicit conversion from value trying to turn it into type
```

Comment:
Do you mean to remove these two lines of old comment?

Reply:
Ping

Reply:
Thanks!